Vagrantfile

```diff
@@ -73,6 +73,10 @@ Vagrant.configure(2) do |config|
     config.vm.box_url = "http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_opensuse-13.2-x86_64_chef-provisionerless.box"
     suse_common config
   end
+  config.vm.define "sles-12" do |config|
+    config.vm.box = "idar/sles12"
```

Comment:
both sles boxes from vagrant images have a DVD repository or none.. which is completely weird to me.

Regarding the licensing situation, what should I look out for? http://demeter.uni-regensburg.de/SLES12-x64/DVD1/ lists a GPL copyright...